spec/asyncapi.md

```diff
@@ -1351,15 +1351,18 @@ Field Name | Type | Description
 <a name="tagObjectName"></a>name | `string` | **Required.** The name of the tag.
 <a name="tagObjectDescription"></a>description | `string` | A short description for the tag. [CommonMark syntax](https://spec.commonmark.org/) can be used for rich text representation.
 <a name="tagObjectExternalDocs"></a>externalDocs | [External Documentation Object](#externalDocumentationObject) | Additional external documentation for this tag.
+<a name="tagObjectValues"></a>values | [`string`] | The value(s) for the tag.
```

Comment:
🤔 I'm not sure I like having `values` inside the global Tag object. We only verified this has a use case on servers but how is it going to affect the rest of the spec where Tag is used? Should we have another kind of Tag object just for servers?

Reply:
For other objects it will work the same way as for servers, just the tags will be treated as "metadata", with which you will be able to filter certain elements in the UI or via the CLI (of course you will have to implement this). Currently a tag only has a key (name field) but you can't define certain more specific values for that (we don't want use delimeter or something like). This can be compared to `labels` in Kubernetes where labels are used to filter/create relationships between resources - https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/

Reply:
Btw, I'm not sure if we should call it `values` or rather `value` in singular, even we allow having multiple. It is just a feeling I guess but the plural sounds weird to me in this case.